modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/AssignQualityScore.pm

```diff
@@ -61,7 +62,7 @@ sub fetch_input {
 		$max_quality{$oid} = $sth->fetchrow_arrayref->[0] or $self->warning("Cannot find quality scores in db for homology id $oid");
         $max_quality{$oid} = 0 unless $max_quality{$oid}; # default to 0 like GOC
     }
-	
```

Comment:
I can't do anything about this, it's a strange white space type I cannot seem to replicate.